pipeline/metadata/blockpage.py

```diff
@@ -21,7 +21,7 @@ def _load_signatures(filepath: str) -> Dict[str, re.Pattern]:
   """
   data = pkgutil.get_data(__name__, filepath)
   if not data:
-    raise FileNotFoundError(f"Couldn't find file {filename}")
+    raise FileNotFoundError(f"Couldn't find file {filepath}")
 
   content = data.decode('utf-8')
   lines = content.split('\n')
```

Comment:
You have 3 copies of the file: `data`, `content` and `lines`.
Instead, wrap data in StringIO and read each line

Reply:
done